ui/page/settingsStripeCard/view.jsx

```diff
@@ -410,7 +451,7 @@ class SettingsStripeCard extends React.Component<Props, State> {
 
         {/* initial markup to show while getting information */}
         {currentFlowStage === 'loading' && (
-          <div className="headerCard toConfirmCard">
+          <div className="headerCard getting-card-status__div">
```

Comment:
No need to change, but hope to see less of it ... seen it a few times:

Having `div` as a BEM _element_ name is weird -- seen this in other PRs.  To the reader, it means "all `<div>`s under the `getting-card-status` block".

If the page has its own `stripe-card` class, then this should just be `stripe-card__getting-card-status`